ang/civicase/CustomScrollbar.js

```diff
@@ -0,0 +1,62 @@
+(function (angular, $, _) {
+  var module = angular.module('civicase');
+
+  module.directive('civicaseCustomScrollbar', function ($rootScope, $parse, $timeout) {
+    return {
+      restrict: 'A',
+      link: civicaseCustomScrollbarLink
+    };
+
+    /**
+     * Link function for civicaseCustomScrollbar
+     *
+     * @param {Object} $scope
+     * @param {jQuery} element
+     * @param {Object} attrs
+     */
+    function civicaseCustomScrollbarLink ($scope, element, attrs) {
+      var options = {
+        autoHide: false
+      };
+
+      if (attrs.civicaseCustomScrollbar) {
+        options = $parse(attrs.civicaseCustomScrollbar)();
+      }
+
+      (function init () {
+        initScrollbar(options);
```

Comment:
there is no point passing `options`, given that it's a variable accessible by every function